lib/gooddata/lcm/actions/update_release_table.rb

```diff
@@ -75,18 +75,10 @@ def replace_placeholders(text, placeholders)
         end
 
         def update_release_table(params, placeholders)
-          query = if placeholders[:version] > 1
-                    path = File.expand_path('../../data/update_lcm_release.sql.erb', __FILE__)
-                    default_query = GoodData::Helpers::ErbHelper.template_file(path, placeholders)
-
-                    temp_query = (params.query && params.query[:update]) || default_query
-                    replace_placeholders(temp_query, placeholders)
-                  else
-                    path = File.expand_path('../../data/insert_into_lcm_release.sql.erb', __FILE__)
-                    default_query = GoodData::Helpers::ErbHelper.template_file(path, placeholders)
-                    temp_query = (params.query && params.query.insert) || default_query
-                    replace_placeholders(temp_query, placeholders)
-                  end
+          path = File.expand_path('../../data/insert_into_lcm_release.sql.erb', __FILE__)
+          default_query = GoodData::Helpers::ErbHelper.template_file(path, placeholders)
+          temp_query = (params.query && params.query.insert) || default_query
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)